packages/fxa-auth-client/lib/client.ts

```diff
@@ -2304,10 +2331,48 @@ export default class AuthClient {
     );
   }
 
+  /**
+   * Confirms the code sent to the recovery phone during a sign in flow.
+   *
+   * @param sessionToken The user's current session token
+   * @param code The otp code sent to the user's phone
+   * @param headers
+   */
+  async recoveryPhoneSignInConfirm(
+    sessionToken: string,
+    code: string,
+    headers?: Headers
+  ) {
+    return this.sessionPost(
+      '/recovery_phone/signin/send_code',
```

Comment:
I think this should be 
```suggestion
      '/recovery_phone/signin/confirm',
```